ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs

```diff
@@ -5,53 +5,60 @@
       <h2>{{t 'onboarding.success.titles.connect'}}</h2>
     </:header>
     <:body>
-      <div class='onboarding-quick-setup-success-boundary'>
-        <div class='onboarding-quick-setup-success-sub-title'>
-          <Rose::Icon @name='logo-app-background' @size='extra-large' />
+      <div class='onboarding-quick-setup-success-role'>
+        <span>
```

Comment:
What is the purpose of this `span`?

Reply:
To make the Icon and h3 tag inline with each other. That is what line 300 in `app.scss` is for.